public/app/index.html

```
@@ -82,12 +82,36 @@ <h1 class="hdr-title">VoiceIsolate Pro <span class="hdr-ver">v24.0</span></h1>
     <div class="col-left">
       <!-- Upload -->
       <div class="card upload-card">
+        <!--
+          FIX: fileInput is placed OUTSIDE uploadZone and rendered off-screen.
+          This prevents the uploadZone click-guard (which blocks BUTTON children)
+          from ever blocking programmatic .click() calls on the input.
+          pointer-events is intentionally NOT set to none — programmatic .click()
+          requires the element to be interactable in the DOM.
+        -->
+        <input
+          type="file"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)